src/tests/test_piranha_swift.rs

```diff
@@ -29,12 +29,12 @@ create_rewrite_tests! {
     substitutions = substitutions! {
       "stale_flag_name" => "Premium-Icon"
     },
-    cleanup_comments = true;
+    cleanup_comments = true, delete_file_if_empty= false;
```

Comment:
Why the inconsistent spacing around `=` here? (between this parameter and the previous two) And why isn't that auto-formatted? (I thought we were running an auto-formatter on every commit now)

Reply:
I will add auto-formatting for swift files in a follow up commit.

Reply:
This file isn't swift, though, it's `.rs` :/